### PR TITLE
Skip Rust's escaping in the enum_def macro (#793)

### PR DESCRIPTION
**Pull request by RichardWGNR**
## PR Info

- Closes: [Issue 793](https://github.com/SeaQL/sea-query/issues/793)

- Dependencies: no

- Dependents: no

## New Features

No

## Bug Fixes

Yes. Please see the #793 issue

## Breaking Changes

No

## Changes

No

### CODE REVIEW

**Review by RichardWGNR — COMMENTED** [review]
.

File: sea-query-attr/src/lib.rs
Comment (RichardWGNR) [inline sea-query-attr/src/lib.rs:1]: I could reduce the number of allocations, but that's not the point here.

File: sea-query-attr/src/lib.rs [hunk 1]
```diff
@@ -58,10 +58,18 @@ pub fn enum_def(args: TokenStream, input: TokenStream) -> TokenStream {
...
-                .to_string();
+            let string = {
+                let mut inner = ident
+                    .as_ref()
+                    .expect("#[enum_def] can only be used on structs with named fields")
+                    .to_string();
+
+                if inner.starts_with("r#") {
+                    inner = (&inner[2..]).to_string();
+                }
```
[the remark this answers is not in the document]
Comment (doveq) [reply sea-query-attr/src/lib.rs:69]: hi, how to use IdentExt::unraw ?

**Comment by tyt2y3** [comment]
sea-query-attr is now deprecated. can you open a new PR for sea-query-derive?

**Pull request closed without merging**